docker-compose.yml

```diff
@@ -463,8 +464,8 @@ services:
   ssh-mssql-connection:
     build:
       context: .
-      platforms:
-        - "linux/amd64"
+      # platforms:
```

Comment:
This stopped me from local builds, and AI says this should be backwards compatible with Linux since the host platform is used.